umap/settings/base.py

```diff
@@ -306,3 +306,10 @@
         },
     },
 }
+
+# WebSocket configuration
+
+WEBSOCKET_ENABLED = env.bool("WEBSOCKET_ENABLED", default=False)
+WEBSOCKET_HOST = env("WEBSOCKET_HOST", default="localhost")
+WEBSOCKET_PORT = env.int("WEBSOCKET_PORT", default=8001)
+WEBSOCKET_URI = env("WEBSOCKET_URI", default="ws://localhost:8001")
```

Comment:
The separation between host/port and URI is needed, because one is the used by the browser, and the other by the server. But we could probably merge HOST and PORT.

Reply:
Above I suggested another option: POST/HOST/TLS, and then we can build the final URI for the frontend from those.

Reply:
Let's continue the discussion there (marking it as "resolved" here)